solutions/usaco-916.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: usaco-916
+source: USACO Bronze 2019 February
+title: The Great Revegetation
+author: Ryan Chou
+---
+## Solution 1
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_revegetate_bronze_feb19.html)
+
+## Solution 2
+We can iterate through the favorite pastures, and increment one to the pasture with the greater index, making the result minimal.
+We'll also cycle through the previous favorite pastures to make sure that our new alteration doesn't interfere with any of the other cow's favorite pastures.
+
+## Implementation
+**Time Complexity:** $\mathcal{O}(N^2\log{}N)$
+
+<LanguageSection>
+<PySection>
+
+```py
```

Comment:
small question does this implement sol 1 or sol 2

Reply:
solution 2

Reply:
yeah cool uh maybe indicate that somewhere? not required but it would be nice i guess

Reply:
added it.

Reply:
nice